locksmith/src/config/config.ts

```diff
@@ -77,6 +77,8 @@ const config = {
     exportsBucket: 'uploads',
   },
   recaptchaSecret: process.env.RECAPTCHA_SECRET,
+  gitcoinApiKey: process.env.GITCOIN_API_KEY,
+  gitcoinScorerId: process.env.GITCOIN_SCORER_ID,
```

Comment:
Can you provide an explaination of what that one is?

Reply:
the Gitcoin API key is required to make valid requests to the Passport API. The Scorer ID facilitates custom scoring rules and application-specific identity verification. Will provide more clarity in the comments